src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx

```diff
@@ -5,38 +5,40 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-
+import './saved_object_view.scss';
```

Comment:
NIT: surprised the linter does not complain about this. this import should be at the bottom (or at least after the absolute imports)

Reply:
Ya, you're right, the linter doesn't seem to care about import order. We import the stylesheet before any other imports in [`import_summary`](https://github.com/elastic/kibana/blob/master/src/plugins/saved_objects_management/public/management_section/objects_table/components/import_summary.tsx#L9), while other plugins have the order [differently](https://github.com/elastic/kibana/blob/master/src/plugins/index_pattern_editor/public/components/index_pattern_editor.tsx#L14).